// _@vuepress_core@1.5.4@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/xuzhen/Documents/demo/my-blog/cherish/layouts/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-6deb18a6",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6deb18a6").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-77dc5296",
    path: "/article/about.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-77dc5296").then(next)
    },
  },
  {
    name: "v-8e7b972a",
    path: "/article/life/guzheng/guzheng.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-8e7b972a").then(next)
    },
  },
  {
    name: "v-7df55e8d",
    path: "/article/life/tea/green.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7df55e8d").then(next)
    },
  },
  {
    name: "v-8d555e72",
    path: "/article/life/plant/plant.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-8d555e72").then(next)
    },
  },
  {
    name: "v-944e8756",
    path: "/article/other.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-944e8756").then(next)
    },
  },
  {
    name: "v-3b619bf6",
    path: "/article/web/css/css1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3b619bf6").then(next)
    },
  },
  {
    name: "v-60d74cd7",
    path: "/article/tools/tool.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-60d74cd7").then(next)
    },
  },
  {
    name: "v-74a98a56",
    path: "/article/web/html/html.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-74a98a56").then(next)
    },
  },
  {
    name: "v-718dc976",
    path: "/article/web/html/html5%E6%96%B0%E7%89%B9%E6%80%A7.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-718dc976").then(next)
    },
  },
  {
    path: "/article/web/html/html5新特性.html",
    redirect: "/article/web/html/html5%E6%96%B0%E7%89%B9%E6%80%A7.html"
  },
  {
    path: "/article/web/html/html5新特性.html",
    redirect: "/article/web/html/html5%E6%96%B0%E7%89%B9%E6%80%A7.html"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
